src/vs/platform/environment/node/argv.ts

```diff
@@ -78,6 +78,16 @@ export const OPTIONS: OptionDescriptions<Required<NativeParsedArgs>> = {
 			'telemetry-level': { type: 'string' },
 		}
 	},
+	'chat': {
```

Comment:
We should also add a fig spec for this which is what adds completions to the terminal:

https://github.com/microsoft/vscode/blob/f63842cd575e497503fc0e3ad78e85235a34a0a0/extensions/terminal-suggest/src/completions/code.ts

The format is fairly simple so should mostly be just a bit of copying and pasting.